haiku-apps/wordclock/wordclock-1.0.recipe

```diff
@@ -0,0 +1,44 @@
+SUMMARY="An imprecise clock replicant"
+DESCRIPTION="WordClock is imprecise clock replicant and this show clock with word"
```

Comment:
Thanks for this PR!
There are only a couple of things to change.

1) Please change DESCRIPTON to:
```
DESCRIPTION="WordClock is an imprecise clock replicant which displays the time \
using words instead of numbers."
```

2) Remove the blank line at the end of the recipe.

3) Fix the rdef file by inserting a PATCH function before BUILD:
```
PATCH()
{
	sed -i \
		-e "/major/ s/=.*/= 1,/;" \
		-e "/middle/ s/=.*/= 0,/;" \
		-e "/minor/ s/=.*/= 0,/;" \
		sources/WordClock.rdef
}
```